MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -2,16 +2,15 @@ import WebKit
 
 internal class MiniAppWebView: WKWebView {
 
-    convenience init?(filePath: URL) {
-        let fileURL = filePath.appendingPathComponent("index.html")
-        if !fileURL.isFileURL {
-            return nil
-        }
+    convenience init?(miniAppId: String) {
+        let schemeName = Constants.miniAppSchemePrefix + miniAppId
+        let urlRequest = URLRequest(url: URL(string: schemeName + "://" + Constants.rootFileName)!)
         let config = WKWebViewConfiguration()
         config.preferences.javaScriptEnabled = true
         config.preferences.setValue(true, forKey: "allowFileAccessFromFileURLs")
+        config.setURLSchemeHandler(URLSchemeHandler(), forURLScheme: schemeName)
```

Comment:
Does the `WKWebViewConfiguration` keep a reference to the `URLSchemeHandler`?
Or does the SDK need to keep a reference to keep it alive?

Reply:
WKWebViewConfiguration will hold the reference of URLSchemeHandler. As long as the MiniAppWebView instance is alive, the WKWebViewConfiguration associated with it will also hold the reference for the URLSchemeHandler